sdks/python/apache_beam/transforms/resources.py

```diff
@@ -177,6 +178,20 @@ def get_merged_value(
 ResourceHint.register_resource_hint('minRam', MinRamHint)
 
 
+class CpuCountHint(ResourceHint):
+  """Describes desired hardware accelerators in execution environment."""
+  urn = resource_hints.CPU_COUNT.urn
+
+  @classmethod
+  def get_merged_value(
+      cls, outer_value, inner_value):  # type: (int, int) -> int
+    return ResourceHint._use_max(outer_value, inner_value)
+
+
+ResourceHint.register_resource_hint('cpu_count', CpuCountHint)
+ResourceHint.register_resource_hint('cpuCount', CpuCountHint)
```

Comment:
add a comment `# Alias for interoperability with SDKs preferring camelCase.` on why 2 names for CpuCountHint for readability.

Reply:
Done.